beetsplug/parentwork.py

```diff
@@ -0,0 +1,142 @@
+                            partof=True
+                            while partof:
+                                partof=False
+                                if 'work-relation-list' in work_info['work']:
+                                    for work_father in work_info['work']['work-relation-list']:
+                                        if work_father['type'] == 'parts': 
+                                            if 'direction' in work_father:
+                                                if work_father['direction'] == 'backward':
+                                                    father_id=work_father['work']['id']
+                                                    partof=True
+                                                    work_info=musicbrainzngs.get_work_by_id(father_id, includes=["work-rels", "artist-rels"])
+                            if 'artist-relation-list' in work_info['work']:
+                                for artist in work_info['work']['artist-relation-list']:
+                                    if artist['type']=='composer':
+                                        if artist['artist']['name'] in parent_composer:
+                                            pass
+                                        else: 
+                                            parent_composer.append(artist['artist']['name'])
+                                            parent_composer_sort.append(artist['artist']['sort-name'])
+                            else:
+                                print('no composer')
+                                print('add one at')
+                                print('https://musicbrainz.org/work/'+ work_info['work']['id'])
+                            if work_info['work']['title'] in parent_work:
+                                pass
+                            else: 
+                                parent_work.append(work_info['work']['title'])
+                                if 'disambiguation' in work_info['work']:
+                                    parent_work_disambig.append(work_info['work']['disambiguation'])
+                except musicbrainzngs.musicbrainz.NetworkError:
+                    continue
+                    i=i+1
+                break    
+            
+            item['parent_work']          = parent_work
+            item['parent_work_disambig'] = parent_work_disambig
+            item['work']                 = work
+            item['work_disambig']        = work_disambig
+            item['performer']            = performer
+            item['performer_sort']       = performer_sort
+            item['parent_composer']      = parent_composer
+            item['parent_composer_sort'] = parent_composer_sort
+            
+            item.try_write()
+            item.store(fields=['parent_work','parent_work_disambig','work','work_disambig','performer','performer_sort','parent_composer','parent_composer_sort'])
```

Comment:
If I do this I get: 
Traceback (most recent call last):
  File "/usr/bin/beet", line 11, in <module>
    load_entry_point('beets==1.4.4', 'console_scripts', 'beet')()
  File "/usr/lib/python3.6/site-packages/beets/ui/__init__.py", line 1228, in main
    _raw_main(args)
  File "/usr/lib/python3.6/site-packages/beets/ui/__init__.py", line 1215, in _raw_main
    subcommand.func(lib, suboptions, subargs)
  File "/usr/lib/python3.6/site-packages/beets/ui/commands.py", line 933, in import_func
    import_files(lib, paths, query)
  File "/usr/lib/python3.6/site-packages/beets/ui/commands.py", line 910, in import_files
    session.run()
  File "/usr/lib/python3.6/site-packages/beets/importer.py", line 325, in run
    pl.run_parallel(QUEUE_SIZE)
  File "/usr/lib/python3.6/site-packages/beets/util/pipeline.py", line 445, in run_parallel
    six.reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/usr/lib/python3.6/site-packages/six.py", line 686, in reraise
    raise value
  File "/usr/lib/python3.6/site-packages/beets/util/pipeline.py", line 312, in run
    out = self.coro.send(msg)
  File "/usr/lib/python3.6/site-packages/beets/util/pipeline.py", line 194, in coro
    func(*(args + (task,)))
  File "/usr/lib/python3.6/site-packages/beets/importer.py", line 1398, in plugin_stage
    func(session, task)
  File "/usr/lib/python3.6/site-packages/beets/plugins.py", line 124, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/beetsplug/parentwork.py", line 59, in imported
    self.find_work(task.items)        
  File "/usr/lib/python3.6/site-packages/beetsplug/parentwork.py", line 135, in find_work
    item.store()
  File "/usr/lib/python3.6/site-packages/beets/library.py", line 326, in store
    super(LibModel, self).store(fields)
  File "/usr/lib/python3.6/site-packages/beets/dbcore/db.py", line 381, in store
    tx.mutate(query, subvars)
  File "/usr/lib/python3.6/site-packages/beets/dbcore/db.py", line 693, in mutate
    cursor = self.db._connection().execute(statement, subvals)
sqlite3.InterfaceError: Error binding parameter 0 - probably unsupported type.